CorelDRAWUpdate/CorelDRAWUpdate.munki.recipe

```diff
@@ -37,12 +45,16 @@
 			<key>display_name</key>
 			<string>CorelDRAW Update</string>
 			<key>name</key>
-			<string>%NAME%</string>
+			<string>%NAME%%BASE_VERSION%%ARCHITECTURE%</string>
```

Comment:
This could lead to confusion because name != name. Why not:
```
<key>NAME</key>
<string>CorelDRAWUpdate%BASE_VERSION%%ARCHITECTURE%</string>

<!--- ... -->

<key>name</key>
<string>%NAME%</string>
```

Reply:
I like that idea. That also makes it easier to change the name.